docs/updating/update-to-37.md

```diff
@@ -0,0 +1,48 @@
+---
+category: update-guides
+menu-title: Update to v37.x
+order: 88
+modified_at: 2023-04-03
+---
+
+# Update to CKEditor 5 v37.x
+
+<info-box>
+	When updating your CKEditor 5 installation, make sure **all the packages are the same version** to avoid errors.
+
+	For custom builds, you may try removing the `package-lock.json` or `yarn.lock` files (if applicable) and reinstalling all packages before rebuilding the editor. For best results, make sure you use the most recent package versions.
+</info-box>
+
+## Update to CKEditor 5 v37.0.0
+
+For the entire list of changes introduced in version 37.0.0 see the [release notes for CKEditor 5 v37.0.0](https://github.com/ckeditor/ckeditor5/releases/tag/v37.0.0-rc.0).
```

Comment:
I put a link to `v37.0.0-rc.0`, because tag `v37.0.0` doesn't exist yet and the verifier would yell at me.